### PR TITLE
Improve code to meet 2018 code standards

### PR DESCRIPTION
**Pull request by MathiasReker**
Just some improvements.

How to test: Everything should work the same.

### CODE REVIEW

**Comment by MathiasReker** [comment]
@AllanWLie can you please review it?

File: epay/epay.php [hunk 1]
```diff
@@ -1281,49 +1310,48 @@ private function buildTransactionFormStart()
...
             $transaction = $this->getDbTransactionsByCartId($order->id_cart);
-        }
-
-        if (!$transaction) {
             $html .= 'No payment transaction was found';
+
             return $html;
+        } else {
+            $transaction = $this->getDbTransactionsByOrderId($order->id);
         }
```
Comment (MathiasReker) [inline epay/epay.php]: Is above OK? @AllanWLie

Comment (MathiasReker) [reply epay/epay.php]: See the changes

Comment (MathiasReker) [reply epay/epay.php]: Solved

**Comment by MathiasReker** [comment]
@AllanWLie Can you please review it? /c @storkie

**Comment by MathiasReker** [comment]
wdyt?

**Comment by MathiasReker** [comment]
I improved the code. Can you please review the PR?

**Comment by MathiasReker** [comment]
I did split my PR up, so it is more easy to review. Just ping me when you want me to solve conflicts.

**Comment by Casper-O** [comment]
And you also just send out 20 emails to people following the project...

**Comment by Casper-O** [comment]
In my opinion what you just did, didn’t make it more reviewable. But let that be up to ePay.

I follow projects I use like ePay, and on small important projects like this I have notifications active as it normally only results in few emails each month.

+20 in such a short time is really to much...

**Comment by AllanWLie** [comment]
Hi @MathiasReker 

Thank you for doing a tremendous job. I want nothing more than to accept you pull request (now requests). The issue is time. We do not currently have the resources for testing all the aspects of your changes. I am sure that what you did is done right. But we must test it all before releasing it to all of our users (merchants). I have added the task to our backlog and when we have the time it will be testet and merged.

Again thank you and keep up the good work.

Best regards
Allan - Bambora

**Comment by MathiasReker** [comment]
@AllanWLie I understand. I made the other PR's so you can choose what changes you want. Btw. don't merge this PR, because it makes the module PHP 5.6+ (not compatible with 1.6.x).

**Pull request closed without merging**